Marlin/Regulator.cpp

```diff
@@ -33,4 +69,98 @@ void setOutputPressure(float desired_pressure) {
     DAC_write(MCP4725_I2C_ADDRESS, digital_val);
 }
 
+
+/**
+ * Updates the protection function, called regularly in the main loop.
+ */
+void Regulator__Update() {
+  pressure_protection(current_target_pressure, pressureRegulator());
+}
+
+//===========================================================================
+//============================ Private Functions ============================
+//===========================================================================
+
+/** 
+ * Error handler when marlin detects a missing pressure regulator
+ */
+static void _regulator_sled_removed_error() {
+  if (sled_present == true) {
+    //quickStop();
+    //disable_all_heaters();
+    //disable_all_steppers();
+    SERIAL_PROTOCOLLNPGM(  
+      "Sled not detected. Canceling print, check pneumatics sled.");
```

Comment:
"Pneumatics not detected" might be a more helpful message than "Sled not detected" for the initial sentence